### DISCUSSION TITLE
Stack not working with negative values anymore

### DISCUSSION THREAD

**Discussion by LucasCodr** [chakra-ui/chakra-ui#7875] [Q&A]
Hi! As I upgraded a project to Chakra's latest version, I noticed that stacks with negative values stopped working. The latest implementation of the Stack component is using the flex gap approach instead of the owl selector for spacing, and I'm wondering if this is the intended result, as the types still allow such values: https://github.com/chakra-ui/chakra-ui/blob/f6c3c5b07025e2ddbaaf496ef52be469d865d4f8/packages/components/layout/src/stack/stack.tsx#L38.

Here's an example of how it used to work by using Chakra's playground on different versions:
![CleanShot 2023-07-20 at 21 23 49@2x](https://github.com/chakra-ui/chakra-ui/assets/29847191/2161e91e-cef6-4d1a-8a63-a9df45bf2e3a)
![CleanShot 2023-07-20 at 21 25 30@2x](https://github.com/chakra-ui/chakra-ui/assets/29847191/2d620e09-8afc-4453-993c-2570c049175b)

**Comment by gotnoshoeson** [comment]
Have the same question. Looks like the Avatar Group allows this functionality, but I want to use a custom component to get the same effect.
